app.js

```diff
@@ -140,7 +140,19 @@ function parseStatsdUrl(url) {
  * @return {string[]}
  */
 function tagsToArr (tags) {
-  return _.transform(tags, (arr, value, key) => arr.push(key + ':' + value), []);
+  return _.transform(tags, (arr, value, key) => {
+    if (key === 'path') {
+      value = stripIds(value);
+    }
+    arr.push(key + ':' + value);
+  }, []);
+}
+
+/**
+ * Strip unique ids from a path to avoid a huge number of metrics
+ */
+function stripIds(value) {
+  return value.replace(/\d+/g, '');
```

Comment:
Replace with something like `:id` rather than `''`. Is it possible to match only numbers following a `/`, or would that not catch all the cases?